src/test/java/com/microsoft/azure/spring/data/cosmosdb/core/DocumentDbTemplateIT.java

```diff
@@ -180,7 +181,7 @@ public void testCountByQuery() {
 
         final Criteria criteria = Criteria.getUnaryInstance(CriteriaType.IS_EQUAL, "firstName",
                 Arrays.asList(TEST_PERSON_2.getFirstName()));
-        final DocumentQuery query = new DocumentQuery(criteria);
+        final DocumentQuery query = new DocumentQuery(criteria, Sort.unsorted());
```

Comment:
How about a constructor with default sort parameter? Better solution is a builder.